integrations/datadog/.port/resources/blueprints.json

```diff
@@ -0,0 +1,228 @@
+        "monitorType": {
+          "type": "string",
+          "title": "Monitor Type"
+        },
+        "tags": {
+          "type": "array",
+          "title": "Tags"
+        },
+        "overallState": {
+          "type": "string",
+          "title": "Overall state",
+          "enum": ["Alert", "Ignored", "No Data", "OK", "Skipped", "Unknown", "Warn"],
+          "enumColors": {
+            "Alert": "red",
+            "Ignored": "darkGray",
+            "No Data": "lightGray",
+            "OK": "green",
+            "Skipped": "yellow",
+            "Unknown": "purple",
+            "Warn": "orange"
+          }
+        },
+        "priority": {
+          "type": "string",
+          "title": "Priority"
+        },
+        "thresholds": {
+          "type": "object",
+          "title": "Thresholds"
+        },
+        "createdBy": {
+          "type": "string",
+          "title": "Creator"
+        },
+        "createdAt": {
+          "title": "Created At",
+          "type": "string",
+          "format": "date-time"
+        },
+        "updatedAt": {
+          "title": "Updated At",
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {}
+  },
+  {
+    "identifier": "datadogSlo",
+    "description": "This blueprint represents a datadog SLO",
+    "title": "Datadog SLO",
+    "icon": "Datadog",
+    "schema": {
+      "properties": {
+        "tags": {
+          "type": "array",
+          "title": "Tags"
+        },
+        "sloType": {
+          "title": "Type",
+          "type": "string"
+        },
+        "description": {
+          "title": "Description",
+          "type": "string"
+        },
+        "warningThreshold": {
+          "title": "Warning Threshold",
+          "type": "string"
+        },
+        "targetThreshold": {
+          "title": "Target Threshold",
+          "type": "string"
+        },
+        "createdAt": {
+          "title": "Created At",
+          "type": "string",
+          "format": "date-time"
+        },
+        "updatedAt": {
+          "title": "Updated At",
+          "type": "string",
+          "format": "date-time"
+        },
+        "createdBy": {
+          "title": "Creator",
+          "type": "string"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "monitors": {
+        "target": "datadogMonitor",
+        "title": "SLO Monitors",
+        "description": "The monitors tracking this SLO",
+        "required": false,
+        "many": true
+      },
+      "services": {
+        "target": "datadogServiceDefinition",
+        "title": "Services",
+        "description": "The services tracked by this SLO",
+        "required": false,
+        "many": true
+      }
+    }
+  },
+  {
+    "identifier": "datadogServiceDefinition",
```

Comment:
rename to `datadogServiceCatalog`

Also, reorder this so that it comes before the SLO blueprint